### ISSUE TITLE
missing alpine linux dependency py-pillow

### ISSUE THREAD

**Issue by fflo** [VGoshev/seafile-docker#2]
Hi Vladimir,

can you please add the package py-pillow to your image?
Without this package trying to add an avatar to an account, seafile crashes with the following message:
```
Page unavailable
Sorry, but the requested page is unavailable due to a server hicup.
Our engineers have been notified, so check back later.
```

I was able to resolve this issue running `apk add python2-pillow`.

**Comment by VGoshev** [comment]
Ouch, thanks for report, I'll fix it in couple of minutes

**Comment by VGoshev** [comment]
Fixed with commit 30de55f077cc29c1be3ef331eb70df1c44d894ca

**Comment by fflo** [comment]
thank you! 👍
